Extension/src/LanguageServer/extension.ts

```diff
@@ -1199,9 +1200,23 @@ async function handleCrashFileRead(crashDirectory: string, crashFile: string, er
         data = data.substring(0, 8191) + "…";
     }
 
-    console.log(`Crash call stack:\n${data}`);
     logCppCrashTelemetry(data, addressData);
 
+    if (data !== prevCppCrashData) {
+        prevCppCrashData = data;
+
+        const settings: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration("C_Cpp", null);
+        if (util.getNumericLoggingLevel(settings.get<string>("loggingLevel")) >= 1) {
+            console.log(`Crash call stack:\n${data}`);
+            if (lines.length >= 6) {
+                const out: Logger = getOutputChannelLogger();
+                out.appendLine(localize({ key: "crash.callstack.available", comment: [ "{0} is the process name: either cpptools or cpptools-srv"]},
+                    "A {0} crash call stack for use in bug reporting is available in the 'Console' tab after using the 'Toggle Developer Tools' command.",
```

Comment:
@bobbrow Would you mind if the crash stacks were logged directly to the C/C++ output log instead of console.log?  I think we'd get a lot more stacks from users that way.  Talking to Sean about it, he thought you may have had a reason we shouldn't do that?  If we do that, also, should it be Error or Debug level?

Reply:
I didn't weigh in on the decision at all.  I'm ok with logging to the Output window with "Error" level.

Reply:
(just don't focus the Output automatically)

Reply:
@bobbrow Okay, sounds good -- I remember talking about it a long time ago.

Reply:
Ok. I don't remember so that means I didn't have a strong opinion.

Reply:
I put it in it's own channel to avoid the issue with it being interleaved/hidden between the list of compiler args to cpptools-srv.